scripts/rtma3d/exrtma3d_rhist.ksh

```diff
@@ -133,13 +139,11 @@ do
 
    case $file in
       perm)   hpssdir=$hpssdir0
-              tarfile=com_rtma3d_${PDYn1}_${cycm3}z-${cycm1}z.tar;;
-
+              tarfile=com_rtma3d_${tarcyc}_${cycm3}z-${cycm1}z.tar;;
```

Comment:
@EdwardColon-NOAA I'm curious why these lines are showing as changed in the PR? The tarfiles are already defined based on tarcyc in all of my parallels.

Reply:
It seems this is replicating the changes made in PR#54: https://github.com/NOAA-EMC/NOAA_3drtma/pull/54/files